http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -644,14 +644,14 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
       ).interruptAfter(config.appMonitorConfig.updateApp.interruptAfter).compile.lastOrError
     } yield appOk.isDone
 
-  private[util] def getHelmAuthContext(clusterName: AKSClusterName,
+  private[util] def getHelmAuthContext(clusterName: String,
```

Comment:
nit: I suggest leaving this as `AKSClusterName` for type-safety and readability. I realize this means you'll need to construct `AKSClusterName` in a few places to satisfy the argument. You could add a convenience method to `AKSCluster` if it's helpful to do the translation, something like:
```scala
final case class AKSCluster(name: String, tags: Map[String, Boolean]) {
  def asClusterName: AKSClusterName = AKSClusterName(name)
}
```